src/perception/PoseEstimatorModule.cpp

```diff
@@ -1,4 +1,4 @@
-#include <aikido/perception/RcnnPoseModule.hpp>
+#include <aikido/perception/PoseEstimatorModule.hpp>
```

Comment:
Nit: Our convention is to use `"..."` for AIKIDO headers instead of `<...>`.

Reply:
changed.